Tools/LyTestTools/ly_test_tools/o3de/pipeline_utils.py

```diff
@@ -459,8 +461,10 @@ def validate_log_output(
         failure_cb()
 
     # Assert no unexpected lines found and all expected queries found
-    assert unexpectedly_found == [], f"Unexpected line(s) were found in the log run: {unexpectedly_found}"
-    assert expected_queries == [], f"Expected query(s) were not found in the log run: {expected_queries}"
+    if not unexpectedly_found == []:
+        raise exceptions.LyTestToolsException(f"Unexpected line(s) were found in the log run: {unexpectedly_found}")
```

Comment:
These look like test assertions as well? If they are then they also shouldn't be LTT exceptions